test/functional/test_runner.py

```diff
@@ -255,7 +255,8 @@
 DISABLED_SCRIPTS = []
 
 # Place EXTENDED_SCRIPTS first since it has the 3 longest running tests
-ALL_SCRIPTS = EXTENDED_SCRIPTS + BASE_SCRIPTS + DISABLED_SCRIPTS + USBDEVICE_SCRIPTS
+# ALL_SCRIPTS = EXTENDED_SCRIPTS + BASE_SCRIPTS + DISABLED_SCRIPTS + USBDEVICE_SCRIPTS
+ALL_SCRIPTS = ['wallet_multiwallet.py'] * 50 + ['wallet_multiwallet.py --usecli'] * 5
```

Comment:
This variables is used differently. You need to say

```python
test_list = ['wallet_multiwallet.py'] * 50 + ['wallet_multiwallet.py --usecli'] * 5
```

in `main` right before `run_tests(...)` invoked.

Reply:
Thanks, will remember that.